sqlitecluster/SQLitePeer.cpp

```diff
@@ -72,8 +72,7 @@ SQLitePeer::PeerPostPollStatus SQLitePeer::postPoll(fd_map& fdm, uint64_t& nextA
         switch (socket->state.load()) {
             case STCPManager::Socket::CONNECTED: {
                 // socket->lastRecvTime is always set, it's initialized to STimeNow() at creation.
```

Comment:
NAB suggest updating to mention that we don't check for `socket->lastSendTime` and why